velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -913,5 +913,64 @@ TEST_F(DateTimeFunctionsTest, fromUnixtime) {
       fromUnixTime(0, "yyyy-MM-dd HH:II"), "Specifier I is not supported");
 }
 
+TEST_F(DateTimeFunctionsTest, makeYMInterval) {
+  const auto getYMInterval = [](const std::optional<int32_t>& ymInterval) {
+    return ymInterval.has_value()
+        ? std::make_optional(INTERVAL_YEAR_MONTH()->valueToString(*ymInterval))
+        : std::nullopt;
+  };
+  const auto fromYearAndMonth = [&](const std::optional<int32_t>& year,
+                                    const std::optional<std::int32_t>& month) {
+    auto result = evaluateOnce<int32_t, int32_t>(
+        "make_ym_interval(c0, c1)",
+        {year, month},
+        {INTEGER(), INTEGER()},
+        std::nullopt,
+        {INTERVAL_YEAR_MONTH()});
+    return getYMInterval(result);
```

Comment:
I suggest the following:

```
VELOX_CHECK(result.has_value())
return INTERVAL_YEAR_MONTH()->valueToString(result.value());

```

Reply:
@mbasmanova Seems like it cannot test null input with this change, e.g. `EXPECT_EQ(fromYearAndMonth(178956971, std::nullopt), std::nullopt);`

Reply:
We do not need to test null inputs. Null inputs are handled by the framework, not the function we are testing. Let's remove tests for null inputs.